app/views/galleries/edit.html.erb

```diff
@@ -7,7 +7,8 @@
   <input type='text' name='gallery[name]' value="<%= @gallery.name %>"><br><br>
 
   <label for="gentrycost">Entry Cost (in USD):</label>
-  <input type='text' name='gallery[entry_cost]' value="<%= @gallery.entry_cost %>"><br><br>
+  <!-- <input type='text' name='gallery[entry_cost]' value="<%= @gallery.entry_cost %>"><br><br> -->
+  <input type='number' name='gallery[entry_cost]' min="0" step="0.01" value="<%= @gallery.entry_cost %>"><br><br>
```

Comment:
Nice change, this means we won't have to add any error handling as it is handled by the form.